tests/test_utils/mock_process.py

```diff
@@ -85,3 +84,15 @@ def __next__(self):
 
     def __iter__(self):
         yield from self.iterable
+
+
+class MockConnectionCursor(BaseMockConnectionCursor):
+    def __init__(self, *args, **kwargs):
+        super().__init__()
+        self.iterable = [(1, 1), (2, 2)]
```

Comment:
I think you can just do this

```suggestion
class MockConnectionCursor(BaseMockConnectionCursor):
    iterable = [(1, 1), (2, 2)]
```

Reply:
Good catch.  Unnecessary args, kwargs arguments have been removed from `mock_process.py`.